flocker/control/test/test_httpapi.py

```diff
@@ -242,6 +242,76 @@ def test_container_name_collision_different_node(self):
         """
         return self._container_name_collision_test(self.NODE_A, self.NODE_B)
 
+    def test_create_container_with_environment(self):
+        """
+        A valid API request to create a container including environment
+        variables results in an updated configuration.
+        """
+        saving = self.persistence_service.save(Deployment(
+            nodes={
+                Node(hostname=self.NODE_A),
+                Node(hostname=self.NODE_B),
+            }
+        ))
+
+        environment = {
+            u'USER': u'webserver',
```

Comment:
Maybe remove or change the `USER` env variable which seems a little contrived (as mentioned in previous review)